tests/test_query_manager.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+
+from ..intermine import query_manager as qm
+qm.save_mine_and_token('flymine', 'k136n1HfFd31n6O4han1')
```

Comment:
Unfortunately we can't use flymine as a test mine.

1. It's a live InterMine used by people. 
2. FlyMine's model might change
3. FlyMine might not live forever.

Instead, do the thing I suggested. Make a dummy mine in your code that has hardcoded valid tokens, returns good and bad results, etc.

This will require that you refactor your code.